### ISSUE TITLE
wp_enqueue_script was called incorrectly

### ISSUE THREAD

**Issue by ziodave** [insideout10/wordlift-plugin#1204]
Please describe the issue here:

```
[17-Nov-2020 10:31:27 UTC] PHP Notice:  wp_enqueue_script was called <strong>incorrectly</strong>. Scripts and styles should not be registered or enqueued until the <code>wp_enqueue_scripts</code>, <code>admin_enqueue_scripts</code>, or <code>login_enqueue_scripts</code> hooks. This notice was triggered by the <code>wl_enabled_blocks</code> handle. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 3.3.0.) in /var/www/html/wordpress/wp-includes/functions.php on line 5225
[17-Nov-2020 10:31:27 UTC] PHP Stack trace:
[17-Nov-2020 10:31:27 UTC] PHP   1. {main}() /var/www/html/index.php:0
[17-Nov-2020 10:31:27 UTC] PHP   2. require() /var/www/html/index.php:17
[17-Nov-2020 10:31:27 UTC] PHP   3. require_once() /var/www/html/wordpress/wp-blog-header.php:13
[17-Nov-2020 10:31:27 UTC] PHP   4. require_once() /var/www/html/wordpress/wp-load.php:42
[17-Nov-2020 10:31:27 UTC] PHP   5. require_once() /var/www/html/wp-config.php:14
[17-Nov-2020 10:31:27 UTC] PHP   6. include_once() /var/www/html/wordpress/wp-settings.php:377
[17-Nov-2020 10:31:27 UTC] PHP   7. run_wordlift() /var/www/html/wp-content/plugins/wordlift/wordlift.php:523
[17-Nov-2020 10:31:27 UTC] PHP   8. Wordlift->__construct() /var/www/html/wp-content/plugins/wordlift/wordlift.php:470
[17-Nov-2020 10:31:27 UTC] PHP   9. Wordlift->load_dependencies() /var/www/html/wp-content/plugins/wordlift/includes/class-wordlift.php:751
[17-Nov-2020 10:31:27 UTC] PHP  10. wp_enqueue_script() /var/www/html/wp-content/plugins/wordlift/includes/class-wordlift.php:1344
[17-Nov-2020 10:31:27 UTC] PHP  11. _wp_scripts_maybe_doing_it_wrong() /var/www/html/wordpress/wp-includes/functions.wp-scripts.php:340
[17-Nov-2020 10:31:27 UTC] PHP  12. _doing_it_wrong() /var/www/html/wordpress/wp-includes/functions.wp-scripts.php:67
[17-Nov-2020 10:31:27 UTC] PHP  13. trigger_error() /var/www/html/wordpress/wp-includes/functions.php:5225
```

Change

```
		wp_localize_script( 'wl_enabled_blocks', 'wlEnabledBlocks', $enabled_blocks );
		wp_enqueue_script( 'wl_enabled_blocks' );
```

to

```
		add_action( 'wp_enqueue_scripts', function () use ( $enabled_blocks ) {
			wp_localize_script( 'wl_enabled_blocks', 'wlEnabledBlocks', $enabled_blocks );
			wp_enqueue_script( 'wl_enabled_blocks' );
		} );
```

Please provide additional data about your WordPress install:
[ ] WordPress version: 5.5.1
[ ] WordLift plugin version: 3.27.6

**Comment by naveen17797** [comment]
Fixed on #1203

**Comment by naveen17797** [comment]
Fixed on #1203
